Authentication_and_Authorization/hello-security/login-controller.php

```diff
@@ -6,19 +6,47 @@
 <body>
 <a href="index.php"> Take me home, country roads ... </a> <br>
 <?php
+
 include "db-connection.php";
+
+function getHashedPassword($conn, $email){
+
+    $sqlQueryForPassword = "SELECT password FROM person WHERE email = :email;";
+
+    $getHashedPassword = $conn->prepare($sqlQueryForPassword);
+    $getHashedPassword->bindParam(':email', $email);
+    $getHashedPassword->execute() or die ("Not valid credentials. There is no user with this email.");
```

Comment:
You may want to change the error. This way currently the user knows that the email he is trying to log in with is not existing in the system. This could be considered security sensitive information that we may want to prevent from being shared.